app/models/peoplesoft_bursar/credit.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+module PeoplesoftBursar
+  class Credit < Amount
```

Comment:
Since this can be confusing, can we add a comment to the class, maybe something along the lines of "A credit from the library to a patron. This means from the library's perspective credits are always negative." (this is my understanding, feel free to re-word or clarify)

Reply:
@maxkadel I added a comment to the code and moved the conversion to it's own line to highlight it.  Please check it out and see if it covers this.